### ISSUE TITLE
AMICI setup failed! - but why?

### ISSUE THREAD

**Issue by thomassligon** [ICB-DCM/AMICI#235]
In amici_interface_matlab.cpp, lines 541-543 should be changed to make the error message more meaningful.
These lines
```
    } catch(...) {
        amici::errMsgIdAndTxt("AMICI:mex:setup","AMICI setup failed!");
    }
```
should be changed to
```
    } catch(const std::exception& ex) {
        amici::errMsgIdAndTxt("AMICI:mex:setup","AMICI setup failed!\n(%s)",ex.what());
    }
```

**Comment by thomassligon** [comment]
While we are at it, we should also improve the preceding message (lines 537-539):
These lines
```
        } catch (...) {
            amici::errMsgIdAndTxt("AMICI:mex", "Unknown internal error occured");
        }
```
should be changed to
```
        } catch (const std::exception& ex) {
            amici::errMsgIdAndTxt("AMICI:mex", "Unknown internal error occured\n(%s)",ex.what());
        }
```

**Comment by FFroehlich** [comment]
true! this is already fixed in #222
